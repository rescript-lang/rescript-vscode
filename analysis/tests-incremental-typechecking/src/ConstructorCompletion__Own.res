module WithVariant = {
  type t = One({miss: bool}) | Two(bool)
}

let x = WithVariant.One()
//                      ^com
