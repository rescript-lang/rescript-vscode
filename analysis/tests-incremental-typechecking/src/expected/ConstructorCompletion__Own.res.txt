Complete src/ConstructorCompletion__Own.res 4:24
posCursor:[4:24] posNoWhite:[4:23] Found expr:[4:8->4:25]
Pexp_construct WithVariant
One:[4:8->4:23] [4:23->4:25]
Completable: Cexpression CTypeAtPos()->variantPayload::One($0)
Package opens Pervasives.JsxModules.place holder
Resolved opens 1 pervasives
ContextPath CTypeAtPos()
[{
    "label": "{}",
    "kind": 4,
    "tags": [],
    "detail": "Inline record",
    "documentation": null,
    "sortText": "A",
    "insertText": "{$0}",
    "insertTextFormat": 2
  }]

